designer/client/__tests__/link-create.jest.tsx

```diff
@@ -24,6 +24,36 @@ const rawData = {
   conditions: [],
 };
 
+const rawDatab = {
```

Comment:
is b meant to be on the end?

Reply:
I havent finished this yet, the PR is still marked as draft